src/__screenshot_tests__/inline-screenshot-test.tsx

```diff
@@ -8,7 +8,7 @@ test('Inline ', async () => {
 });
 
 test('Inline wrap', async () => {
-    await openStoryPage({id: 'layout-inline--wrap'});
+    const page = await openStoryPage({id: 'layout-inline--wrap'});
```

Comment:
Using Puppeteer's page instead of our wrapper was causing screenshots to be taken too soon (before the story finishes loading)